todo_app/templates/index.html

```diff
@@ -12,7 +12,16 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+        {% for item in items %}
+          <li>{{ item.id}} - {{item.status}} - {{item.title}}</li>
+        {% endfor %}
       </ul>
+      <h2>Add an item</h2>
+      <form method="post" action="/addItem" >
+        <label for="itemName">Item name:&nbsp;</label>
```

Comment:
Rather than adding a space here we'd find it more flexible to add some styling - e.g. we could add `style="padding-right: 5px;"` to our label and vary the value to control the spacing between these elements